### ISSUE TITLE
Colab is hung

### ISSUE THREAD

**Issue by prakashbhattgit** [googlecolab/colabtools#1451]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:
while running a model.fit my colab is hanging

- Describe the expected behavior:

while runnign below it is stuck at 

Train on 625 samples, validate on 625 samples
Epoch 1/50
160/625 [======>.......................] - ETA: 1s - loss: 0.3578 - accuracy: 0.8438 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 2/50
160/625 [======>.......................] - ETA: 1s - loss: 0.3664 - accuracy: 0.8438 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 3/50
160/625 [======>.......................] - ETA: 1s - loss: 0.3369 - accuracy: 0.8813 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 4/50
145/625 [=====>........................] - ETA: 1s - loss: 0.4149 - accuracy: 0.8345 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 5/50
160/625 [======>.......................] - ETA: 1s - loss: 0.3550 - accuracy: 0.8438 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 6/50
160/625 [======>.......................] - ETA: 1s - loss: 0.3455 - accuracy: 0.8188 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 7/50
160/625 [======>.......................] - ETA: 1s - loss: 0.3233 - accuracy: 0.8438 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 8/50
145/625 [=====>........................] - ETA: 1s - loss: 0.2378 - accuracy: 0.8897 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00Epoch 9/50
160/625 [======>.......................] - ETA: 0s - loss: 0.3190 - accuracy: 0.8625

code

  # Train model
    start_time = time.time()
    training_results = resnet_model.fit(
            train_data_x,
            train_data_y,
            validation_data=(X_val,y_val),
            batch_size=batch_size,
            epochs=epochs, 
            callbacks = callbacks,
            #callbacks = [es,mc],
            steps_per_epoch = steps_per_epoch,
            validation_steps = val_step,
            verbose=verbose)
    execution_time = (time.time() - start_time)/60.0
    print("Training execution time (mins)",execution_time)

- The web browser you are using (Chrome, Firefox, Safari, etc.):

- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):

**Comment by metrizable** [comment]
Since this issue was filed a while ago, if you are still experiencing the issue, please re-open and provide a minimal reproducible notebook.
